### ISSUE TITLE
cant recover an old deck

### ISSUE THREAD

**Issue by mmiat** [HearthSim/Hearthstone-Deck-Tracker#3684]
### Expected Behavior
if i resume and old/archived deck, i want to restart tracking it

### Actual Behavior
HDT create a new deck

### Steps to reproduce behavior
not sure: create a deck in HS, it's created also in HDT. delete it from HS, then recreate with same name or importing from HDT, HDT create a new deck

### Log/Screenshots
i attach screenshot
![screenshot 2018-05-14 22 51 42](https://user-images.githubusercontent.com/11910225/40022844-71975cd0-57c9-11e8-9c7b-5e34ac325269.png)
now i've renamed new one as "Silence Priest 2018" to seee the new one

also, i cant' associate new deck to the older one, if i select the older when playing, HDT move to the new one. can i create a new version of old (v4 in my screenshot), move games and use the old deck?

**Comment by Eleven-MXI** [comment]
Did you ever find a solution to this?

**Comment by mauro-miatello** [comment]
> Did you ever find a solution to this?

No, I didn't, but honestly recently I'm playing without tracking decks
